StarryExpanse/recog.py

```diff
@@ -0,0 +1,22 @@
+import subprocess
+import sys
+
+cog_command = []
+if sys.platform == 'darwin':
+    cog_command = ['cog.py']
+else:
+    cog_command = ['python', 'C:\Python36\Scripts\cog.py']
```

Comment:
Don't you need to prefix the second string with 'r', like `r'C:\Python36\Scripts\cog.py'` to disable the backslash from being an escape sequence?